packages/swapper/src/swappers/ChainflipSwapper/utils/helpers.ts

```diff
@@ -0,0 +1,25 @@
+import { type AssetId, type ChainId } from '@shapeshiftoss/caip'
+
+import type { ChainflipSupportedChainId } from '../constants'
+import {
+  assetGasLimits,
+  ChainflipSupportedAssetIdsByChainId,
+  ChainflipSupportedChainIds,
+} from '../constants'
+
+export const isSupportedChainId = (chainId: ChainId): chainId is ChainflipSupportedChainId => {
+  return ChainflipSupportedChainIds.includes(chainId as ChainflipSupportedChainId)
+}
+
+export const isSupportedAssetId = (
+  chainId: ChainId,
+  assetId: AssetId,
+): chainId is ChainflipSupportedChainId => {
+  return ChainflipSupportedAssetIdsByChainId[chainId as ChainflipSupportedChainId]!.includes(
+    assetId,
+  )
+}
+
+export const getGasLimit = (chainflipAsset: string) => {
```

Comment:
This won't generalize and should be programmatic. Since we're talking sends, allowance isn't a problem - there will never be allowance required (which, on a tangent, means we should be cognicent in ensuring allowance step is never triggered in swapper)